src/components/SignUpForm/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,102 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
you don't need both these snapshots as they're basically the same

Reply:
that's right, when I edit anything, there would be two snapshots edit